cmake/onnxruntime_framework.cmake

```diff
@@ -57,7 +57,7 @@ if (onnxruntime_ENABLE_TRAINING OR onnxruntime_ENABLE_TRAINING_OPS)
     target_include_directories(onnxruntime_framework PUBLIC ${MPI_CXX_INCLUDE_DIRS})
   endif()
 endif()
-if (onnxruntime_ENABLE_TRAINING)
+if (NOT onnxruntime_MINIMAL_BUILD)
```

Comment:
You are adding garbage to the core framework. 
1. This Aten fallback feature would never work on Windows
2. This Aten fallback feature requires python

We deliver ORT to all platforms, for many many programming languages.  Why do they need this dlpack library?

Reply:
+1. Please exclude dlpack from regular builds.

Reply:
The ATen fallback works on Windows, I tested it on my Win machine. And yes currently ATen fallback is designed for scenarios using Python, but logically it also works without Python by using C++ LibTorch. Since it's for Python scenarios for now, I've changed the code to add it when ENABLE_PYTHON is ON.